src/test/java/com/google/api/codegen/testdata/simplecompute_config.baseline

```diff
@@ -300,6 +300,8 @@ interfaces:
   collections:
   - name_pattern: projects/{project}/dummyObjects/{dummyObject}
     entity_name: dummyObject
+  - name_pattern: projects/{project}/dummyObjects/{resource}
```

Comment:
Changes in this file are just a direct, autogenerated consequence of the changes in compute.v1.json.